doc/examples/github_app.ml

```diff
@@ -31,14 +31,21 @@ let dockerfile ~base =
 let weekly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 7) ()
 
 (* Map from Current.state to CheckRunStatus *)
-let github_check_run_status_of_state = function
-  | Ok _              -> Github.Api.CheckRunStatus.v ~url (`Completed `Success) ~description:"Passed"
-  | Error (`Active _) -> Github.Api.CheckRunStatus.v ~url `Queued
-  | Error (`Msg m)    -> Github.Api.CheckRunStatus.v ~url (`Completed (`Failure m)) ~description:m
+let github_check_run_status_of_state ?job_id = function
+  | Ok _              -> Github.Api.CheckRunStatus.v ~url ?job_id (`Completed `Success) ~description:"Passed"
+  | Error (`Active _) -> Github.Api.CheckRunStatus.v ~url ?job_id `Queued
+  | Error (`Msg m)    -> Github.Api.CheckRunStatus.v ~url ?job_id (`Completed (`Failure m)) ~description:m
+
+let check_run_status x =
+  let+ md = Current.Analysis.metadata x
+  and+ state = Current.state x in
+  match md with
+  | Some { Current.Metadata.job_id; _ } -> github_check_run_status_of_state ?job_id state 
+  | None -> github_check_run_status_of_state state
 
 let pipeline ~app () =
   let dockerfile =
-    let+ base = Docker.pull ~schedule:weekly "ocaml/opam:alpine-3.12-ocaml-4.08" in
+    let+ base = Docker.pull ~schedule:weekly "ocaml/opam:alpine-3.13-ocaml-4.08" in
```

Comment:
Bumping this because we aren't publishing `alpine-3.12` since 4 months ago and it was causing issues running these examples.